backends/README.md

```diff
@@ -36,18 +36,20 @@ X seconds (though, it can send them per second if you need it to).
 
 3. Netdata can filter metrics (at the chart level), to send only a subset of the collected metrics.
 
-4. Three modes of operation (for all backends):
+4. Netdata supports three modes of operation for all backends. **If you can't decide which mode to use, 
```

Comment:
I think we should suggest this:

- `as-collected` sends to backends the metrics as they are collected, in the units they are collected. So, counters are sent as counters and gauges are sent as gauges, much like all data collectors do. For example, to calculate CPU utilization in this format, you need to know how to convert kernel ticks to percentage.

- `average` sends to backends normalized metrics from the netdata database. In this mode, all metrics are sent as gauges, in the units netdata uses. This abstracts data collection and simplifies visualization, but you will not be able to copy and paste queries from other sources to convert units. For example, CPU utilization percentage is calculated by netdata, so netdata will convert ticks to percentage and sent the average percentage to the backend.

Time-series databases suggest to collect the raw values (`as-collected`).

If you plan to invest on building your monitoring around a time-series database, and you already know (or you will invest in learning) how to convert units and normalize the metrics at Grafana or other visualization tools, we suggest to use `as-collected`.

If, on the other hand, you just need long term archiving of netdata metrics, and you plan to mainly work with netdata, we suggest to use `average`. It decouples visualization from data collection, so it will generally be a lot simpler.